packages/extension/src/components/organisms/NFTDetails/NFTDetails.tsx

```diff
@@ -0,0 +1,43 @@
+import React, { FC } from 'react';
+
+import CloseIcon from '@mui/icons-material/Close';
+import { AppBar, IconButton, List, Toolbar, Typography } from '@mui/material';
+
+import { NFTData } from '@gemwallet/constants';
+
+import { NFTImage, NFTListItem } from '../../atoms';
+
+interface NFTDetailsProps {
+  NFTData: NFTData;
+  handleClose: () => void;
+}
+
+export const NFTDetails: FC<NFTDetailsProps> = ({ NFTData, handleClose }) => {
+  return (
+    <>
+      <AppBar sx={{ position: 'relative' }}>
+        <Toolbar>
+          <IconButton
+            edge="start"
+            color="inherit"
+            aria-label="close"
+            onClick={handleClose}
+            style={{ cursor: 'pointer' }}
+            data-testid="close-button"
+          >
+            <CloseIcon />
+          </IconButton>
+          <Typography sx={{ ml: 2, flex: 1 }} variant="h6" component="p">
+            NFT Details
+          </Typography>
+        </Toolbar>
+      </AppBar>
+      <List sx={{ width: '100%', wordBreak: 'break-word' }}>
+        <NFTImage imageURL={NFTData.image} />
+        <NFTListItem primary="Token ID" secondary={NFTData.NFTokenID} />
```

Comment:
Actually, now that I see it, I am thinking it's maybe not necessary to create that extra `NFTListItem` component... It doesn't simplify much and hardcoding the style directly into `NFTListItem` is not optimized.

We could just extract the `ListItem` style into its own const that we create outside the component and reuse it.

```jsx
const listItemStyle = { padding: "8px 24px" }

export const NFTDetails = () => {
...

return (
...
<ListItem style={listItemStyle}>
     <ListItemText primary={} secondary={} />
</ListItem>
...
```

Sorry for the change :)

Reply:
Done 👍